tests/unit/coordinator/test_list.py

```diff
@@ -228,7 +229,7 @@ def test_api_list_deduplication(backup_manifest: BackupManifest, tmpdir: PathLik
     storage.upload_hexdigest_bytes("FAKEDIGEST", b"fake-digest-data")
 
     list_request = ListRequest(storage="x")
-    list_response = list_backups(req=list_request, json_mstorage=multi_rohmu_storage)
+    list_response = list_backups(req=list_request, json_mstorage=multi_rohmu_storage, cache={})
```

Comment:
should there be some test added to make sure cache is used properly?

Reply:
Good point, I've added some positive and negative tests.